pkg/operator/staticpod/controller/staticpodfallback/static_pod_fallback_condition_test.go

```diff
@@ -173,3 +175,38 @@ func newPod(phase corev1.PodPhase, ready corev1.ConditionStatus, revision, name
 
 	return &pod
 }
+
+// we need a deterministic pod lister for tests
+// otherwise the order of the pods will yield different results
+type orderedPodNamespaceLister struct {
+	podLister corev1listers.PodNamespaceLister
```

Comment:
why not deeply nest without name? Then you don't need the Get func

Reply:
First interface ive seen :)